### DISCUSSION TITLE
Vercel - Deployment Failed

### DISCUSSION THREAD

**Discussion by ghost** [neuefische/questions#344] [Web] [answered]
Hi all, 

As I finished my first PR the Vercel showed me this: 

> Vercel – capstone-project-movies-series-app — Deployment has failed

I've tried to delete the project and connect it again but now after I pushed my project I get two errors from Vercel: 

> Vercel – capstone-project-movies-series-app — Deployment has failed
> Vercel – capstone-project-movies-series-app-o88n — Deployment has failed

I've reached for help but nobody could find an answer.  

Can someone help me with this issue?

**Comment by stefankorder** [comment]
Hi Georgios,
can you post the log of vercel and a link to your repo?
TY

**Reply by ghost** [reply]
Hi Stefan, 

sure:

https://github.com/Natrosvili/Capstone-Project-movies-series-app/pull/13

<img width="1440" alt="Screenshot 2022-11-30 at 14 54 46" src="https://user-images.githubusercontent.com/100809251/204814448-d41a779d-f4be-4c44-bbfc-5acf94ce9ddc.png">

**Comment by actyralf** [comment, marked as answer]
It seems that your `/pages/footer/index.js` is empty and does not return a valid React component. Every file (or directory) you put in `/pages` MUST return a valid component which represents the page. `header` and `footer` sound if they should be components your want to place on pages later, so please move them to the `/components` directory.
